.gitmodules

```diff
@@ -33,3 +33,6 @@
 [submodule "src/isa-l"]
 	path = src/isa-l
 	url = https://github.com/ceph/isa-l
+[submodule "boost"]
+	path = boost
```

Comment:
That was my intention, I managed to confuse myself by I think doing it wrong initially and then not cleaning up correctly.